tsconfig.json

```diff
@@ -2,25 +2,42 @@
     "compileOnSave": false,
     "include": ["projects/**/*"],
     "exclude": ["projects/**/*.spec.ts", "projects/demo-integrations", "**/schematics/**/*"],
+    "angularCompilerOptions": {
+        "preserveWhitespaces": false,
+        "annotateForClosureCompiler": false,
+        "skipTemplateCodegen": true,
+        "strictMetadataEmit": true,
+        "fullTemplateTypeCheck": true,
+        "strictInjectionParameters": true,
+        "enableResourceInlining": true
+    },
     "compilerOptions": {
         "baseUrl": "./",
         "skipLibCheck": true,
         "downlevelIteration": true,
         "outDir": "./dist",
-        "sourceMap": true,
+        "skipDefaultLibCheck": true,
+        "sourceMap": false,
+        "inlineSourceMap": true,
+        "inlineSources": true,
         "declaration": false,
         "module": "esnext",
         "moduleResolution": "node",
         "experimentalDecorators": true,
         "resolveJsonModule": true,
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
+        "preserveConstEnums": true,
         "importHelpers": true,
         "strict": true,
         "noFallthroughCasesInSwitch": true,
         "noUnusedLocals": true,
         "noImplicitReturns": true,
         "noUnusedParameters": true,
+        "noEmitOnError": true,
```

Comment:
These are all set to `true` by `"strict": true`

Reply:
But `default` property is false by default (and has no effect with strict)

<img width="924" alt="image" src="https://user-images.githubusercontent.com/12021443/153863776-af027437-3aa9-4ae0-aeca-872b4581f4a9.png">

Reply:
fixed